docs/android/gradle.md

```diff
@@ -1,205 +1,200 @@
+    ```groovy
+        dependencyResolutionManagement {
+            repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+            repositories {
+                ...
+                maven {
+                    url "https://artifactory.productscience.app/releases"
+                }
             }
         }
-    }
     ```
 
-=== "Kotlin DSL"
-    ```kotlin title="settings.gradle.kts"
-    ...
-    dependencyResolutionManagement {
+=== "Kotlin"
+    ```kotlin
         ...
-        repositories {
-            maven {
-                url = uri("https://artifactory.productscience.app/releases")
+        dependencyResolutionManagement {
+            repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+            repositories {
+                ...
+                maven {
+                    url = uri("https://artifactory.productscience.app/releases")
+                }
             }
         }
-    }
     ```
 
-If this block does not exist, you can create it at the top level of your **settings.gradle** file.
+**Note:** If you use `RepositoriesMode.FAIL_ON_PROJECT_REPOS` mode you may experience a failure when one or more modules define their own repositories in their `build.gradle` files. Switching the mode to `RepositoriesMode.PREFER_SETTINGS` may solve this problem. Alternatively, you may add the `productscience` maven repository url to these modules `build.gradle` files. You can also use the method defined in Case 1 above instead of defining them in `settings.gradle`.
 
+## 4. Add Product Science plugin to classpath
 
-### Case 2: Using the `allprojects` block in the root **build.gradle** file
-Add the repository url to the `allprojects` block in the root **build.gradle** file:
+In the `buildscript` block of the __root__ `build.gradle` file, add the classpaths for the `productscience` `transformer-plugin` and `transformer-instrumentation` artifacts:
 
 === "Groovy"
-    ```groovy title="build.gradle"
-    
-    allprojects {
-        repositories {
-            maven {
-                url "https://artifactory.productscience.app/releases"
+    ```groovy
+        buildscript {
+            repositories { ... }
+            dependencies {
+                ...
+                classpath "com.productscience.transformer:transformer-plugin:0.18.4"
+                classpath "com.productscience.transformer:transformer-instrumentation:0.18.4"
             }
         }
-    }
-    ```  
-
-=== "Kotlin DSL"
-    ```kotlin title="build.gradle.kts"
+        ...
+    ```
 
-    allprojects {
-        repositories {
-            maven {
-                url = uri("https://artifactory.productscience.app/releases")
+=== "Kotlin"
+    ```kotlin
+        buildscript {
+            repositories { ... }
+            dependencies {
+                ...
+                classpath("com.productscience.transformer:transformer-plugin:0.18.4")
+                classpath("com.productscience.transformer:transformer-instrumentation:0.18.4")
```

Comment:
Why to hardcode the version of plugin?

Reply:
I can help to share code that pulls latest plugin version

Reply:
Use android_release() so this example would look like this: classpath("com.productscience.transformer:transformer-plugin:{{ android_release() }}")